telco-core/configuration/reference-crs-kube-compare/optional/other/sctp_module_mc.yaml

```diff
@@ -0,0 +1,25 @@
+# optional
+# count: 1
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
```

Comment:
enforcing the worker role might not work here, as not everyone is going to apply this to all their workers and instead to a more specific role

Reply:
Agreed, custom selector should be allowed here.

Reply:
By that do you mean any label or and value for that key?

Reply:
For now any value for that key